operations/mimir/README.md

```diff
@@ -1,7 +1,7 @@
 # Jsonnet for Grafana Mimir on Kubernetes
 
 This folder contains the Jsonnet for deploying Grafana Mimir in Kubernetes.
-The documentation for the Mimir Jsonnet is published at [https://grafana.com/docs/mimir/next/operators-guide/deploy-grafana-mimir/jsonnet/](https://grafana.com/docs/mimir/next/operators-guide/deploy-grafana-mimir/jsonnet/).
+The documentation for the Mimir Jsonnet is published at [https://grafana.com/docs/mimir/next/manage/deploy-grafana-mimir/jsonnet/](https://grafana.com/docs/mimir/next/manage/deploy-grafana-mimir/jsonnet/).
```

Comment:
Discussed IRL, this should be more generic, point to top of docs so we don't have to maintain.

Reply:
```suggestion
For documentation about Mimir Jsonnet, go to [https://grafana.com/docs/mimir/next/](https://grafana.com/docs/mimir/next/).
```